lnrpc/walletrpc/walletkit.proto

```diff
@@ -962,6 +970,11 @@ message FundPsbtRequest {
 
     // Whether unconfirmed outputs should be used as inputs for the transaction.
     bool spend_unconfirmed = 7;
+
+    //  The scope for the change (if any). If empty, BIP0086 will be used for
```

Comment:
Question for Lightning Labs: with this PR, is it still the idea to keep the 0.15.5 default of p2tr? Because changing the default was the breaking change, and making the change scope configurable doesn't change that.

Of course with this new field, it does becomes relatively easy to adapt existing applications.

Reply:
When you say "keep the 0.15.5 default of p2tr" do you mean for the `FundPsbt` method or in general? We changed the default change address to p2tr everywhere in the codebase (SendCoins RPC, channel open calls, sweeper and so on) and I'm pretty sure we don't want to change those back or make configurable (as that would be a big change threading through the config everywhere).

If you're just talking about the `FundPsbt` method I could see how it might be useful to change the default back to p2wkh once we have this option.

Reply:
Yes, I was talking about just `FundPsbt` and the impact of the default change on external applications.

Reply:
Okay, in this case I would agree with you and argue for defaulting back to p2wkh to not break the RPC for existing users (and do it in this PR as this introduces the means for us to select p2tr change when needed in Taro for example). @Roasbeef do you agree?

Reply:
The solution changed a little (and now defaulting to p2wkh):

https://github.com/lightningnetwork/lnd/pull/7348/files#diff-f92ee065de0a8d5d61de2f8e08f4c31b37611ed54edc4b8522984735d88e7540R75